example/login.js

```diff
@@ -6,13 +6,13 @@ import CustomField from '../src/fields/custom-field';
 import { rules } from './../src/validate';
 import validForm from './../src/validForm';
 
-function MyCustomComponent({ children, input, inlineError, meta, ...props }) {
+function MyCustomComponent({ children, input, errorMessage, meta, ...props }) {
     const id = ('' + Math.random()).slice(2);
     return (
         <div>
             <label htmlFor={id}>{children}</label>
             <input type="text" id={id} {...input} {...props} />
-            <div>{inlineError}</div>
+            <div role="alert" aria-live="assertive">{errorMessage}</div>
```

Comment:
I think its ok to let the user handle everything here. Thats the purpose of the component. If you want to let the component handle things for you, then you can use the labeled field. Well, that's the idea anyways...